### ISSUE TITLE
Player class: create documentation for new prefab items

### ISSUE THREAD

**Issue by vanolson** [uchicago-cs/chiventure#1693]
I recently created new prefab items in `class_prefabs.c`, along with testing, for all of our prefab classes except the NPC ones. Now that these are in our module, we need to document all their details in the wiki. Vincent, Jonathan, and I will add the various attributes of these new items to the wiki, most likely in the Examples Stats and Abilities section, where they will be paired along with the preexisting documentation regarding classes and their skills. We will include information regarding the items' IDs, short descriptions, and long descriptions.

**Comment by jroven** [comment]
Here is the link to the wiki page: [Examples Stats and Abilities](https://github.com/uchicago-cs/chiventure/wiki/Player-Class-~-Example-Stats-and-Abilities)

**Comment by vanolson** [comment]
I added information for wizard, ranger, and druid.

**Comment by Vincent-Zhang4135** [comment]
In the Examples Stats and Abilities Page, I filled in the relevant stats and skills for Basic, and Monk. Currently the NPC classes do not have a skills design, so these classes are not documented with their abilities, but their stats are populated. 

The only thing left to do is to include some more items that the classes can use, which I have communicated with Jonathan to let him do that

**Comment by vanolson** [comment]
Jonathan, Vincent, and I wrote documentation for the new prefab items to the [Examples Stats and Abilities](https://github.com/uchicago-cs/chiventure/wiki/Player-Class-~-Example-Stats-and-Abilities) page on the player class wiki, splitting up the work to each add different items to the page. We added information on the name, short description, and long description, of each item listed in a new **Items** section on the page listed alongside abilities and player statistics. If new items are added in the future, potentially for NPCs, this page will have to be updated.
